lib/ansible/modules/source_control/github_webhook_facts.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
+#
+# Copyright: (c) Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: github_webhook_facts
+short_description: Query information about Github webhooks
+version_added: "2.6"
+description:
+  - "Query information about Github webhooks"
+requirements:
+  - "PyGithub >= 1.3.5"
+options:
+  repository:
+    description:
+      - Full name of the repository to configure a hook for
+    required: true
+  user:
+    description:
+      - User to authenticate to Github as
+    required: true
+  password:
+    description:
+      - Password to authenticate to Github with
+    required: false
+  token:
+    description:
+      - Token to authenticate to Github with
+    required: false
+  github_url:
+    description:
+      - Base URL of the github api
+    required: false
+    default: https://api.github.com
```

Comment:
Does github_url will be different for enterprise use ? If it is going to be common for all then no need to provide this option.

Reply:
Yes, it will be different for GitHub Enterprise.